_studio/mfx_lib/decode/vp9/src/mfx_vp9_dec_decode_hw.cpp

```diff
@@ -416,7 +416,7 @@ mfxStatus VideoDECODEVP9_HW::Init(mfxVideoParam *par)
 #endif
             reallocFrames;
 
-    if (isUseExternalFrames && !reallocFrames)
+    if (isUseExternalFrames)
```

Comment:
need to understand if such correction doesn't harm some use case

Reply:
yes, thanks. 
@onabiull do you know who made the original changes? why reallocFrames not need m_FrameAllocator->SetExternalFramesResponse(&m_response);? 

thanks

Reply:
@xuguangxin I've checked git history and this code was introduced intentionally to handle Dynamic Resolution Change scenarios

Reply:
@mgonchar , thanks for checking. It's for vp9 p frame resolution changing. It may need time to found a decent change for this. Please focus on other changes. I can drop the vp9 related and fix it later.
thanks